src/test.ts

```diff
@@ -0,0 +1,310 @@
+// import "reflect-metadata";
```

Comment:
should we commit this? I wouldn't think so.. ? maybe we can have a look if we can commit without it 😅

Reply:
Yes. I agree. This was just a example for testing.